src/exchange.ts

```diff
@@ -0,0 +1,131 @@
+import BN from "bn.js";
+import { BatchExchangeInstance } from "../types/truffle-typings";
+
+export interface TokenInfo {
+  id: number;
+  symbol: string;
+  decimals: number;
+  address: string;
+}
+
+const MAXU32 = new BN(2).pow(new BN(32)).sub(new BN(1));
```

Comment:
Maybe this could be exported and named more generally like LIQUIDITY_ORDERS_VALID_TO ?